pkg/plugins/platform/local.go

```diff
@@ -266,6 +268,10 @@ func (l *LocalApp) SiteStatus() string {
 	var siteStatus string
 	services := map[string]string{"web": "", "db": ""}
 
+	if !fileutil.FileExists(l.AppRoot()) {
```

Comment:
This only notifies a user if the entire `AppRoot` is missing. The case where just a `.ddev/config.yaml` file is missing is also an issue. It would be great to add another check here (and error return) if the config can't be found.